README.md

```diff
@@ -373,7 +373,7 @@ are all equal and welcome means of contribution. See the
 ## Slack
 
 Get an [invite to our Slack channel](https://join.slack.com/t/cloud-hypervisor/shared_invite/enQtNjY3MTE3MDkwNDQ4LWQ1MTA1ZDVmODkwMWQ1MTRhYzk4ZGNlN2UwNTI3ZmFlODU0OTcwOWZjMTkwZDExYWE3YjFmNzgzY2FmNDAyMjI)
-and [join us on Slack](https://cloud-hypervisor.slack.com/).
+, and [join us on Slack](https://cloud-hypervisor.slack.com/), and [participate our community activities](https://cloud-hypervisor.slack.com/archives/C04R5DUQVBN).
```

Comment:
Please drop the first `and`.

Please change `participate` to `participate in`.

Thanks.

Reply:
done.